### ISSUE TITLE
CLI tool

### ISSUE THREAD

**Issue by nuest** [o2r-project/erc-spec#8]
Create a tool that allows to run an ERC from a command-line interface:

```
erc create /directory /erc-dir
erc package /erc-dir my_research.zip
erc reproduce my_research.zip
```

Do it with `golang` :-).

**Comment by edzer** [comment]
Will this CL tool call the microservices?

**Comment by nuest** [comment]
No, it would be called by the microservices when a new ERC is created. But I am open to use cases where it is the other way around.

**Comment by nuest** [comment]
Sorry @edzer, I completely mixed up the repositories here, In my head I was thinking about https://github.com/o2r-project/containerit/issues/12

So while my comment above is false, the CLI tool I mention here would _still not_ call the microservices but re-create their functionality "offline".
Such a CLI tool _could_ be used by microservices offering a web API so functions are not implemented twice.

Hope this clarifies!
